crates/developer/src/errors.rs

```diff
@@ -0,0 +1,69 @@
+use mcp_core::handler::{ResourceError, ToolError};
```

Comment:
seems like file this should maybe be part of more central library? but i guess reasonable to wait until we have another (although @michaelneale has a new system that would need these i think)

Reply:
Actually after reading, maybe we remove this entirely and only use ToolError from mcp_core